packages/lesswrong/components/LWBackgroundImage.tsx

```diff
@@ -21,7 +22,18 @@ const styles = defineStyles("LWBackgroundImage", (theme: ThemeType) => ({
       right: '0px',
     }
   },
-
+  reviewResultsImage: {
+    position: 'absolute',
+    width: '57vw',
+    maxWidth: '1000px',
+    top: '-70px',
+    right: '-334px',
+    '-webkit-mask-image': `radial-gradient(ellipse at center top, ${theme.palette.text.alwaysBlack} 55%, transparent 70%)`,
+    
+    [theme.breakpoints.up(2000)]: {
```

Comment:
2000 is a weird number, but ok

Reply:
actually it looks like this breakpoint was Wrong and Bad and I don't even know why I thought it was a good idea or remember doing it.